### DISCUSSION TITLE
Option + Arrow as shortcut

### DISCUSSION THREAD

**Discussion by acjacobson** [rxhanson/Rectangle#726] [Q&A] [answered]
I was wondering if it's possible to add support for option key + arrow key shortcuts in Rectangle (like can be done in Specticle). For some reason the app won't let me this up. 

I assume because there is a conflict with other MacOS shortcuts - but Rectangle doesn't seem to mind me overwriting other functionality (like using CMD + arrow key, which is used to go end of line / beginning of line).

**Comment by rxhanson** [comment, marked as answer]
That is a strange discrepancy indeed. One of the frustrating points about using an external library for recording shortcuts. 

Check the box for "Remove shortcut recorder safeguards" in the settings tab. This has tripped up a number of users before, so maybe I will make that box checked by default.

**Reply by acjacobson** [reply]
Thanks so much - this worked perfectly.
